src/NLog.Loki/NLog.Loki.csproj

```diff
@@ -20,7 +20,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="NLog" Version="4.7.15" />
```

Comment:
I'm not sure, but I think I want to keep targetting NLog v5 for a while, so as the prevent from breaking projects running on NLog v4 for now. Let's keep this PR open for a few months, then I'll merge and release a v2 of NLog.Targets.Loki.